tests/test_bokky_poo_bahs_date-time-library.py

```diff
@@ -62,6 +81,28 @@ def test_timestamp_from_date_automated(date_time_library, start_date, end_date):
         current_date += relativedelta(days=+1)
 
 
+@given(
+    timestamp=strategy(
+        "uint256",
+        min_value=get_timestamp_from_date(2020, 1, 1),
+        max_value=get_timestamp_from_date(2050, 1, 1),
+    )
+)
+@settings(max_examples=500, verbosity=Verbosity.quiet)
+def test_property_based_timestamp_from_date(date_time_library, timestamp):
```

Comment:
I think it's better to iterate over dates instead of timestamps: to avoid duplicate checks agains timestamps which belong to the sate date.
If cannot set start/finish _date_ for strategy could set days delta from some hardcoded start date.